### ISSUE TITLE
Add prettier for code formatting

### ISSUE THREAD

**Issue by mikeldking** [ClimbAssist/ClimbAssistUI#15]
I think adding prettier as a dev dependency could really help with consistency. This would be a good first issue for me?

**Comment by buyeno** [comment]
yeah, that's fine it just formats when you commit changes right?

**Comment by mikeldking** [comment]
Right, I can setup a git hook to do that. I typically just have it setup to run on save on my editor.

**Comment by buyeno** [comment]
ya same, I switched to vscode and the prettier formatting is different than atom's defaults so the files that haven't been changed recently aren't consistent 🤷
